### ISSUE TITLE
Font selection

### ISSUE THREAD

**Issue by ronaldtse** [paneron/extension-hls#5]
From @wkwong-ribose 

> For font, Anton said he would see if Paneron can provide a mechanism for the extension to load the font files.

We have to use openly-licensed fonts since this is a public extension.

cc: @strogonoff , @edhenderson

**Comment by strogonoff** [comment]
Correct. Also, I’m still of the opinion that if we can do without custom fonts for now, that would be ideal for UX consistency across Paneron.

**Comment by edhenderson** [comment]
We can load the CSS such that:
- looks for the BSI fonts
- fallback to open source font
- fallback to generic/common font
- fallback to `sans-serif` (which on most computers will be `Arial`, `Helvetica`, or similar)

I don't believe the UX of the app will be drastically affected by a tweak to the font.

**Comment by edhenderson** [comment]
> @edhenderson sounds reasonable to me. We do keep in mind if BSI wants a branded version of Paneron we would need to do some tailoring somewhere, but that's then.

Understood, how likely is this do you think?

**Comment by edhenderson** [comment]
> Note that this method will not fall back to the font provided by Paneron’s stylesheet. Going to check which which font stack is used by Paneron, probably the ideal fallback if BSI fonts are not available.

Thats great, let me know. How are fonts served in Paneron?

**Comment by ronaldtse** [comment]
@edhenderson its a certainty, I just don’t know when would be the latest delivery date for it.
